src/components/Priority/Priority.js

```diff
@@ -0,0 +1,314 @@
+              label: classes.label, // class name, e.g. `classes-nesting-label-x`
+              rail: classes.rail,
+              mark: classes.mark,
+              markLabel: classes.markLabel,
+              offset: classes.offset,
+              track: classes.track,
+            }}
+          />
+        </div>
+        <div className="priority_item">
+          <Icon className='priority_item_icon' name='importance.svg' size='small' />
+          <div className="priority_item_title">Importance</div>
+          <Slider
+            defaultValue={50}
+            getAriaValueText={valuetext}
+            aria-labelledby="discrete-slider-custom"
+            step={25}
+            valueLabelDisplay="auto"
+            marks={marks}
+            classes={{
+              root: classes.root, // class name, e.g. `classes-nesting-root-x`
+              marked: classes.marked,
+              label: classes.label, // class name, e.g. `classes-nesting-label-x`
+              rail: classes.rail,
+              mark: classes.mark,
+              markLabel: classes.markLabel,
+              offset: classes.offset,
+              track: classes.track,
+            }}
+          />
+        </div>
+        <div className="priority_item">
+          <Icon className='priority_item_icon' name='impact.svg' size='small' />
+          <div className="priority_item_title">Impact</div>
+          <Slider
+            defaultValue={50}
+            getAriaValueText={valuetext}
+            aria-labelledby="discrete-slider-custom"
+            step={25}
+            valueLabelDisplay="auto"
+            marks={marks}
+            classes={{
+              root: classes.root, // class name, e.g. `classes-nesting-root-x`
+              marked: classes.marked,
+              label: classes.label, // class name, e.g. `classes-nesting-label-x`
+              rail: classes.rail,
+              mark: classes.mark,
+              markLabel: classes.markLabel,
+              offset: classes.offset,
+              track: classes.track,
+            }}
+          />
+        </div>
+        <div className="priority_item">
+          <Icon className='priority_item_icon' name='effort.png' size='small' />
+          <div className="priority_item_title">Effort</div>
+          <Slider
+            defaultValue={50}
+            getAriaValueText={valuetext}
+            aria-labelledby="discrete-slider-custom"
+            step={25}
+            valueLabelDisplay="auto"
+            marks={marks}
+            classes={{
+              root: classes.root, // class name, e.g. `classes-nesting-root-x`
+              marked: classes.marked,
+              label: classes.label, // class name, e.g. `classes-nesting-label-x`
+              rail: classes.rail,
+              mark: classes.mark,
+              markLabel: classes.markLabel,
+              offset: classes.offset,
+              track: classes.track,
+            }}
+          />
+        </div>
```

Comment:
there is a LOT of duplicate code here, so that is a clear signal to refactor this a bit, so that the code is more editable. I know this will be new, and gets into more "codey" code stuff, but it's important for maintaining quality code. 

I would like to recommend you to read this article, so that you can make sense of all the places that we use the very important `.map` function all across the react components, and throughout the js code.
https://reactjs.org/docs/lists-and-keys.html

That article can guide the way to cutting this code down so that we only have to write the complex 'Slider' code ONCE in the code, and reuse it many times. I want you to give a try at figuring out an approach to this.

Reply:
One easy way to think about this is making the our usage of slider into its own component, and then importing here and referencing it as a component like `<PrioritySlider />` or whatever name

Reply:
cool, the Slider code looks like a super useful one

Reply:
https://vegibit.com/rendering-a-list-with-react/